{
  "uid" : "83a38e46b12c9095",
  "name" : "User tries to pay a cart with two RPTs that exists already in GPD in invalid state",
  "fullName" : "User pays a payment carts from existing payment position via nodoInviaCarrelloRPT: User tries to pay a cart with two RPTs that exists already in GPD in invalid state",
  "historyId" : "3d26ee416cc3affe0103c3d8164ab3c5",
  "time" : {
    "start" : 1749735228490,
    "stop" : 1749735234905,
    "duration" : 6415
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
  "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.4/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.4/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.4/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.4/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
    "steps" : [ {
      "name" : "Given systems up",
      "time" : {
        "start" : 1749735228490,
        "stop" : 1749735229578,
        "duration" : 1088
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Given a cart of RPTs non-multibeneficiary",
      "time" : {
        "start" : 1749735229578,
        "stop" : 1749735229579,
        "duration" : 1
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And a single RPT of type BBT with 1 transfers of which none are stamps",
      "time" : {
        "start" : 1749735229579,
        "stop" : 1749735229579,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And a single RPT of type BBT with 1 transfers of which none are stamps",
      "time" : {
        "start" : 1749735229579,
        "stop" : 1749735229579,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And an existing payment position related to first RPT with segregation code equals to 48 and state equals to DRAFT",
      "time" : {
        "start" : 1749735229579,
        "stop" : 1749735230213,
        "duration" : 634
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "456bf534f56e5ab0",
        "name" : "Step: [an existing payment position related to first RPT with segregation code equals to 48 and state equals to DRAFT] - Sent request",
        "source" : "456bf534f56e5ab0.txt",
        "type" : "text/plain",
        "size" : 1260
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "When the user tries to pay a cart of RPTs on EC website",
      "time" : {
        "start" : 1749735230214,
        "stop" : 1749735231090,
        "duration" : 876
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "d49926356eafb390",
        "name" : "Step: [the user tries to pay a cart of RPTs on EC website] - Sent request",
        "source" : "d49926356eafb390.txt",
        "type" : "text/plain",
        "size" : 15123
      }, {
        "uid" : "11d3e344ee8e2a34",
        "name" : "Step: [the user tries to pay a cart of RPTs on EC website] - Received response",
        "source" : "11d3e344ee8e2a34.txt",
        "type" : "text/plain",
        "size" : 409
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "Then conversion to new model fails in wisp-converter",
      "time" : {
        "start" : 1749735231090,
        "stop" : 1749735232022,
        "duration" : 932
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "ba9d1035b4d262e0",
        "name" : "Step: [conversion to new model fails in wisp-converter]: on sessionId: [6dd830bf-7a9e-4517-afaf-78ecf5646408] - Received response",
        "source" : "ba9d1035b4d262e0.txt",
        "type" : "text/plain",
        "size" : 5807
      }, {
        "uid" : "208d4d2968c85626",
        "name" : "Received response",
        "source" : "208d4d2968c85626.attach",
        "type" : "text/html",
        "size" : 5677
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "And the KO receipt is sent",
      "time" : {
        "start" : 1749735232022,
        "stop" : 1749735234904,
        "duration" : 2882
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.4/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.4/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "9d1edca0cad45a61",
        "name" : "Step: [the KO receipt is sent]: on IUV: [857146432491941] - Received response",
        "source" : "9d1edca0cad45a61.txt",
        "type" : "text/plain",
        "size" : 54938
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 8,
    "attachmentsCount" : 6,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "tag",
    "value" : "nodo_invia_carrello_rpt"
  }, {
    "name" : "tag",
    "value" : "runnable"
  }, {
    "name" : "tag",
    "value" : "unhappy_path"
  }, {
    "name" : "feature",
    "value" : "User pays a payment carts from existing payment position via nodoInviaCarrelloRPT"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "runnable", "nodo_invia_carrello_rpt", "unhappy_path" ]
  },
  "source" : "83a38e46b12c9095.json",
  "parameterValues" : [ ]
}